virttest/utils_test/libvirt.py

```diff
@@ -3760,3 +3761,354 @@ def check_dumpxml(vm, content, err_ignore=False):
                 raise exceptions.TestFail("Expected '%s' was not found in "
                                           "%s's xml" % (content, vm.name))
     return True
+
+
+#cgroup related functions
+class CgroupTest(object):
```

Comment:
Maybe separate it into one singleton file , such as CgroupUtils.py since there are too many stuff in libvirt.py file

Reply:
done, now it's "virttest/libvirt_cgroup.py", cgroup special utils is virttest/staging/utils_cgroup.py, this file is mainly for libvirt, so I named it libvirt_cgroup.py